packages/react-shopify-app-route-propagator/package.json

```diff
@@ -24,7 +24,7 @@
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-shopify-app-route-propagator/README.md",
   "peerDependencies": {
     "@shopify/app-bridge": "^0.7.3",
```

Comment:
since this will be a breaking change, should I take this opportunity to update app-bridge to at least 1.0.0?